python/paddle/v2/framework/framework.py

```diff
@@ -562,3 +562,9 @@ def __init__(self, block, shape, dtype, **kwargs):
 # program is a global instance.
 g_main_program = Program()
 g_startup_program = Program()
+
+def default_startup_program():
+    return g_startup_program
+
+def default_main_program():
```

Comment:
Why we need to wrap the main program into a global function....

Reply:
Here explains: https://github.com/PaddlePaddle/Paddle/issues/5614

Reply:
okay..